frb_codegen/src/lib.rs

```diff
@@ -64,10 +64,17 @@ pub fn frb_codegen_multi(
     all_symbols: &[String],
 ) -> anyhow::Result<()> {
     info!("Phase: Validate config(s)");
-    assert!(all_configs
-        .iter()
-        .enumerate()
-        .all(|(index, config)| config.block_index == BlockIndex(index)));
+    assert!(all_configs.iter().enumerate().all(|(i, config)| {
+        if config.block_index == BlockIndex(i) {
+            true
+        } else {
+            panic!(
+                "config index mismatch: order index({}) != block_index({})",
+                i, config.block_index
+            );
+        }
+    }));
```

Comment:
uber-nit: that `true` seems a bit redundant now, shall we, e.g.

```rust
for config in all_configs.iter().enumerate() {
  assert_eq!(config.block_index, BlockIndex(i), 
      "config index mismatch: order index({}) != block_index({})",       i, config.block_index);
}
```

Reply:
Done in [c30d805](https://github.com/fzyzcjy/flutter_rust_bridge/pull/1124/commits/c30d805d8e91d6fa53b26dc0fbab608a01a1a674). Also, I uber-nit the message, since the default assertion message is already clear.
Now when panic occurs, it would be something like:
```log
2023/03/16 10:31:45 [ERROR] panicked at 'assertion failed: `(left == right)`
  left: `BlockIndex(0)`,
 right: `BlockIndex(2)`: order index(0) != block_index(2)', frb_codegen\src/lib.rs:68:9
```